candidates/views/api.py

```diff
@@ -433,3 +433,8 @@ class ComplexPopoloFieldViewSet(viewsets.ModelViewSet):
     queryset = extra_models.ComplexPopoloField.objects.order_by('id')
     serializer_class = serializers.ComplexPopoloFieldSerializer
     pagination_class = ResultsSetPagination
+
+class PersonRedirectViewSet(viewsets.ModelViewSet):
```

Comment:
Good point. I'm afraid I didn't know about this when first adding the API, which is why read-only-ness is enforced by `DEFAULT_PERMISSION_CLASSES = 'candidates.api_permissions.ReadOnly'` and a lot of these serializers use `ModelViewSet`.